tests/testsuite/build_scripts_multiple.rs

```diff
@@ -470,17 +469,18 @@ fn custom_build_script_second_index_script_failed() {
         .file("build2.rs", "fn main() { std::process::exit(101); }")
         .build();
 
-    p.cargo("build -v")
+    p.cargo("check -v")
```

Comment:
Mind moving this to its own commit.  If there are any other `build`s that you added, they should probably all be changed to `check`.  We try to avoid `build` as much as possible in CI because it is slow.  This is something I overlooked before.

Reply:
Sorry, I did this by mistake 😅 I was checking whether `check` produces error or not